_config.yml

```diff
@@ -10,6 +10,10 @@ date_format: '%B %-d, %Y'
 
 search_briefs_url: issue-briefs/?brief_type=
 
+search_briefs_details_url: /issue-briefs/?details.
+
+search_analysis_url: /analysis/?categories=
```

Comment:
Can we change this variable name to indicate that it's for the categories? And then update in the HTML accordingly.

Reply:
Fixed! Changed `search_analysis_url` to `search_categories_url`.